app/src/features/Incentives/modals/IncentiveTasksListModal/IncentiveTasksListModal.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Modal } from "antd";
+import { IncentiveTasksList } from "../../components/IncentiveTasksList/IncentiveTasksList";
+import "./incentiveTasksListModal.scss";
+
+interface IncentiveTasksListModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+export const IncentiveTasksListModal: React.FC<IncentiveTasksListModalProps> = ({ isOpen, onClose }) => {
```

Comment:
Update the folder name `app/src/features/Incentives` ->  `app/src/features/incentivization`